libiocage/lib/JailConfig.py

```diff
@@ -41,7 +41,7 @@ class JailConfig(dict, object):
 
     Special Properties:
 
-        Special properties are 
+        Special properties are
```

Comment:
The author suddenly stopped writing *couch*

Reply:
Haha